odoo/addons/base/models/res_company.py

```diff
@@ -72,6 +72,8 @@ def _get_default_favicon(self, original=False):
     # logo_web: do not store in attachments, since the image is retrieved in SQL for
     # performance reasons (see addons/web/controllers/main.py, Binary.company_logo)
     logo_web = fields.Binary(compute='_compute_logo_web', store=True, attachment=False)
+    report_display_logo = fields.Binary(string="Report Logo", compute='_compute_report_display_logo', inverse='_inverse_report_display_logo')
+    report_logo = fields.Binary(string="Report Logo Store")
```

Comment:
"Report Logo Store" means a shop which sells report logos... :) Remove the string definition and replace the one of the "report_display_logo" field by "Displayed Logo For Reports".